pynest/nest/spatial/hl_api_spatial.py

```diff
@@ -82,8 +84,17 @@ def n(dimension):
         """
         return CreateParameter('distance', {'dimension': dimension})
 
+    @staticmethod
+    def getInstance():
+        if DistanceParameter.__distanceParam:
+            return DistanceParameter.__distanceParam
+        else:
+            DistanceParameter.__distanceParam = DistanceParameter()
+            return DistanceParameter.__distanceParam
+
 
-distance = DistanceParameter()
+def distance():
```

Comment:
Could probably add a docstring to `distance` as well.

Reply:
docstring has been added